### PR TITLE
LRQA-70726

### PR DESCRIPTION
**Pull request by leticiachiu**
Related Issue: https://issues.liferay.com/browse/LRQA-70726

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 1]
```diff
@@ -710,7 +706,11 @@ definition {
 			locator1 = "FormFields#TOOLBAR_ANY_ITEM_BUTTON",
 			value1 = "Preformatted Text");
 
-		SelectFrame.selectSecondFrame();
+		SelectFrameTop();
+
+		FormPortlet.waitFormRenderer(text = "Save");
+
+		//SelectFrame.selectSecondFrame();
```
[the remark this answers is not in the document]
Comment (leticiachiu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase]: i removed here: https://github.com/liferay-forms/liferay-portal/pull/1007/commits/0ee0c35773d2c7f072fbbbee6cdb1114044f7ef0

**Pull request closed without merging**